web/components/issues/issue-layouts/list/base-list-root.tsx

```diff
@@ -103,6 +108,7 @@ export const BaseListRoot = observer((props: IBaseListRoot) => {
         storeType={storeType}
         addIssuesToView={addIssuesToView}
         isCompletedCycle={isCompletedCycle}
+        handleOnDrop={handleOnDrop}
```

Comment:
Is it for when we are changing the grouping?

Reply:
yes, this is the method that is called on drop with all the data